CoreFeatures/RemoteManagement/FeatureLoadStarterActor/src/main/java/info/smart_tools/smartactors/remote_management/feature_load_starter_actor/wrapper/SetParamsToLoadFromFileWrapper.java

```diff
@@ -26,4 +26,12 @@ void setObservedDirectory(final String directory)
      */
     String getFeatureLocation()
             throws ReadValueException;
+
+    /**
+     *
+     * @return
+     * @throws ReadValueException
+     */
```

Comment:
This javadoc gives to users insufficient information about the method.

Reply:
Descriptions were added to wrapper methond